src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -643,4 +651,23 @@ private static void match(String text, String propertyName, String pattern,
       }
     }
   }
+
+  /**
+   * Deletes temporary directory.
+   *
+   * @param configuration Hadoop Configuration.
+   * @param dir directory to delete
+   */
+  public static void deleteTemporaryDirectory(Configuration configuration, String dir) {
+    Path path = new Path(dir);
+    try {
+      FileSystem fs = path.getFileSystem(configuration);
+      if (fs.exists(path)) {
+        fs.delete(path, true);
+        LOG.debug("Deleted temporary directory '{}'", path);
+      }
+    } catch (IOException e) {
+      LOG.warn("Failed to delete temporary directory '{}': {}", path, e.getMessage());
```

Comment:
Please log an error. If it happens in pipeline we start collecting buckets and we want to know about it sooner than later

Reply:
Will do.